c2cgeoportal/views/pdfreport.py

```diff
@@ -155,7 +157,7 @@ def get_report(self):
                 "outputformat": "gml3",
                 "request": "GetFeature",
                 "typeName": self.layername,
-                "featureid": self.layername + "." + id,
+                "featureid": ",".join(features_ids),
```

Comment:
Usually for these kind of reports a datasource with a query in the template is used:
https://github.com/camptocamp/baselstadt_mapbs/blob/master/print/print-apps/mapbs/Hochstammobstbaeume_Baeume.jrxml#L10-L19

We could do the same for multiple feature ids but we have to make sure that there is no SQL injection possible.